src/Application/Features/Applicant/Command/CreateApplicant/CreateApplicantCommandHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MRA.Jobs.Application.Contracts.Applicant.Commands;
+
+namespace MRA.Jobs.Application.Features.Applicant.Command.CreateApplicant;
+using MRA.Jobs.Domain.Entities;
+
+public class CreateApplicantCommandHandler : IRequestHandler<CreateApplicantCommand, Guid>
+{
+    private readonly IMapper _mapper;
+    private readonly IApplicationDbContext _context;
+
+    public CreateApplicantCommandHandler(
+        IApplicationDbContext context,
+        IMapper mapper)
+    {
+        _mapper = mapper;
+        _context = context;
+        /*_dataTime = dateTime;
```

Comment:
If you don't need this code don't comment it, Just delete it.

Reply:
fixed